crates/ruff_python_formatter/src/statement/stmt_try.rs

```diff
@@ -56,40 +57,64 @@ impl FormatNodeRule<StmtTry> for FormatStmtTry {
         let StmtTry {
             body,
             handlers,
-            orelse: _,
-            finalbody: _,
+            orelse,
+            finalbody,
             is_star,
             range: _,
         } = item;
 
         let comments_info = f.context().comments().clone();
         let mut dangling_comments = comments_info.dangling(item);
 
-        (_, dangling_comments) = format_case(item, CaseKind::Try, None, dangling_comments, f)?;
+        let try_last_suite_in_statement =
+            handlers.is_empty() && orelse.is_empty() && finalbody.is_empty();
+        (_, dangling_comments) = format_case(
```

Comment:
I don't think that's technically possible. A try statement must always have an `except` handler or a `finally` body. That's why it should be fine to assume `false` for `Try`.